examples/modeling_features/07-imported-plies.py

```diff
@@ -0,0 +1,246 @@
+        geom_mesh = cad.visualization_mesh.to_pyvista()
+        plotter.add_mesh(geom_mesh, color=colors[index], opacity=0.1)
+        edges = geom_mesh.extract_feature_edges()
+        plotter.add_mesh(edges, color="white", line_width=4)
+        plotter.add_mesh(edges, color="black", line_width=2)
+    plotter.camera_position = CAMERA_POSITION
+    return plotter
+
+
+plotter = plotter_with_all_geometries([triangle_surf_cad, top_surf_cad])
+plotter.show()
+
+# %%
+# Map Imported Plies onto a solid mesh
+# ------------------------------------
+#
+# An external solid mesh is loaded now to map the imported plies
+# onto the solid model. The next figure shows the imported solid mesh
+# and the imported plies.
+solid_mesh_file = get_example_file(ExampleKeys.BASIC_FLAT_PLATE_SOLID_MESH_CDB, WORKING_DIR)
+imported_solid_model = model.create_imported_solid_model(
+    name="Imported Solid Model",
+    external_path=solid_mesh_file,
+    format="ansys:cdb",
+)
+imported_solid_model.import_initial_mesh()
+plotter = plotter_with_all_geometries([triangle_surf_cad, top_surf_cad])
+plotter.add_mesh(imported_solid_model.solid_mesh.to_pyvista(), show_edges=True, opacity=0.5)
+plotter.show()
+
+# %%
+# Add a mapping object to link the imported plies with the solid model.
+# In this example, all imported plies are mapped in one go.
+# The remaining elemental volume and elements which do not intersect
+# with the imported plies are filled with a void and filler material,
+# respectively.
+imported_solid_model.create_layup_mapping_object(
+    name="Map imported plies",
+    use_imported_plies=True,  # enable imported plies
+    select_all_plies=True,  # select all plies
+    entire_solid_mesh=True,
+    scale_ply_thicknesses=False,
+    void_material=void_material,
+    delete_lost_elements=False,
+    filler_material=filler_material,
+    rosettes=[model.rosettes["12"]],
+)
+model.update()
+
+# %%
+# Show the imported ply geometries and mapped plies on the solid model.
+# Note that the analysis plies are not yet directly accessible via
+# the API of the imported solid model. Also, elemental data such as
+# thicknesses are not yet implemented for imported plies.
+plotter = plotter_with_all_geometries([triangle_surf_cad, top_surf_cad])
+for imported_ply in [imported_ply_triangle, imported_ply_top]:
+    for pp in imported_ply.imported_production_plies.values():
+        for ap in pp.imported_analysis_plies.values():
+            plotter.add_mesh(ap.solid_mesh.to_pyvista(), show_edges=True, opacity=1)
+plotter.add_mesh(mesh=imported_solid_model.solid_mesh.to_pyvista(), show_edges=False, opacity=0.2)
+plotter.show()
+
+# %%
+# The imported solid model can be passed to Mechanical or MAPDL to run an analysis
+# as shown in the examples :ref:`pymechanical_solid_example` and
+# :ref:`pymapdl_workflow_example`.
+#
+# Due to the lack of support of visualization of imported plies, the model
+# can be loaded in ACP standalone and reviewed there.
```

Comment:
```suggestion
```
Can be moved up to where we mention the visualization is not yet supported (see comment there).